data/json/items/melee.json

```diff
@@ -1443,7 +1443,24 @@
     "price_postapoc": 100,
     "material": "steel",
     "weight": 10,
-    "cutting": 1
+    "cutting": 1,
+    "qualities": [ [ "BUTCHER", 9 ], [ "CUT_FINE", 1 ], [ "CUT", 1 ] ]
```

Comment:
Seems too small and/or awkward to qualify for any of those qualities.

Reply:
I respectfully disagree, I've used razor blades as a tool plenty of times.